ci/aks/aks-cluster.lokocfg.envsubst

```diff
@@ -118,3 +118,9 @@ component "httpbin" {
 component "experimental-istio-operator" {
   enable_monitoring = true
 }
+
+component "headlamp" {
+  ingress {
+    host = "headlamp.${var.cluster_name}.${var.aws_dns_zone}"
```

Comment:
Do we want to use production certificates on AKS? If so, why?

Reply:
Oops, good catch.